generator/osm2meta.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "indexer/feature_data.hpp"
+#include "indexer/contacts_processor.hpp"
```

Comment:
We sort includes by alphabet inside one include block.

Reply:
renamed to `validate_and_format_contacts.hpp` and sorted includes alphabetially.